core/templates/dev/head/pages/exploration-editor-page/modal-templates/post-publish-modal.template.html

```diff
@@ -66,8 +66,10 @@ <h3>Awesome!</h3>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
     <div class="oppia-share-publish-link">
-      Here's your exploration link:
+      Here's your exploration link:<br>
+      <span style="visibility:hidden" ng-if="copied==true">&nbsp Copied &nbsp<i class="fa fa-check" aria-hidden="true"></i></span>
```

Comment:
Avoid using inline style. Make it `ng-show="copied"` to achieve "display hidden".
Do we need `&nbsp`?

Reply:
I will change ng-if to ng-show and commit changes. &nbsp is only used to make UI better.